SocialCareCaseViewerApi.Tests/V1/Gateways/MashReferralGatewayTests/UpdateReferralTests.cs

```diff
@@ -73,7 +73,7 @@ public void UpdatingMashReferralThrowsMashReferralStageMismatchExceptionWhenRequ
             Action act = () => _mashReferralGateway.UpdateReferral(request, mashReferral.Id);
 
             act.Should().Throw<MashReferralStageMismatchException>()
-                .WithMessage($"Referral {mashReferral.Id} is in stage \"{mashReferral.Stage}\", this request requires the referral to be in stage \"initial\"");
+                .WithMessage($"Referral {mashReferral.Id} is in stage '{mashReferral.Stage}', this request requires the referral to be in stage 'INITIAL'");
```

Comment:
this (and all similar lines) is a change made for cosmetic purposes, I'm assuming

Reply:
I was checking actual requests and the output it produced and noticed that we also got escape characters in the output, this change addresses that issue.